### ISSUE TITLE
Interaction with aurelia-validation

### ISSUE THREAD

**Issue by ctjhoa** [jdanyow/aurelia-breeze#32]
Hi there,

I'm wondering how do you want to integrate this plugin with `aurelia-validation` (as it's seems to be the next aurelia big step).

Do you have any plan to merge both validation systems (breeze & aurelia-validation) ? Because I've started a [JSData v3 plugin](https://github.com/ctjhoa/aurelia-jsdata) and I don't want to duplicate the validation system in model & controller layer.

**Comment by jdanyow** [comment]
yep- the plan is to create a breeze implementation of the Validator interface in aurelia-validatejs.  I haven't done it yet because we're still working through API changes in aurelia-validation (still very alpha right now).
